components/cloudtables/actions/delete-row/delete-row.mjs

```diff
@@ -0,0 +1,37 @@
+// legacy_hash_id: a_bKijbM
```

Comment:
You'll want to remove this comment. This is for older actions that we programmatically converted, and we use this comment for analytics, but it's not relevant here.